assets/items/box/box.item.yaml

```diff
@@ -0,0 +1,25 @@
+name: Box
+
+image:
+    image: box.png
+    image_size: [32, 32]
+
+kind: !BreakableBox
+    hits: 1
+
+    damage: 20
+
+    hurtbox:
+        size: [36, 52]
+        offset: [0, 0]
+
+    item:
+        name: Bottle
+
+        image:
+            image: bottle.png
+            image_size: [11, 31]
```

Comment:
I'm not sure how this gets implemented but we could import the item yaml file here as we do with fighters in the level. @zicklag might know a little better about this.

Reply:
Yes, I think we should specify the path to the item here, and then in item asset loader we'd use code similar to the code here, to load the item handles from the path:

https://github.com/fishfolks/punchy/blob/d8bfbdcda79e576025c1e0367307a601844a122d/src/assets.rs#L191-L199